### PR TITLE
get started jupyter notebook

### PR DESCRIPTION
**Pull request by SilasK**
Hy 
I tryed out your exciting wrapper for scopus. 
I found some small bugs and was able to correct them. As I work often in Jupyter notebook I copied the conted of the read me in a Get started notebook. 

However the rendering of the reports could be improved. 

Kind regards
Silas

### CODE REVIEW

File: scopus/scopus_api.py [hunk 1]
```diff
@@ -599,7 +599,7 @@ def __init__(self, ISSN, refresh=False):
         SNIP = get_encoded_text(self.results,
                                 'entry/SNIPList/SNIP')
         SNIP_year = self.results.find('entry/SNIPList/SNIP', ns)
-        if SNIP_year:
+        if SNIP_year is not None:
```
Comment (jkitchin) [inline scopus/scopus_api.py]: Why do you think this is necessary? It looks like the same logic.

File: scopus/scopus_author.py [hunk 2]
```diff
@@ -109,8 +109,7 @@ def __init__(self,
         level = number of * to print in __str__.
         """
 
-        if isinstance(author_id, int):
-            author_id = str(author_id)
+        author_id = str(author_id)
```
Comment (jkitchin) [inline scopus/scopus_author.py]: Why do you think this is necessary? You could end up with 'None' as an author name in this case.

**Comment by jkitchin** [comment]
The reports are not designed for the IPython notebooks. For them, you would want to write different report functions. The functions create org-mode formatted output.

File: scopus/scopus_api.py (hunk 1 above)
Comment (SilasK) [inline scopus/scopus_api.py]: I got an error-message, staiting I should write is not None.
For isbn you already have is not None

File: scopus/scopus_author.py (hunk 2 above)
Comment (SilasK) [inline scopus/scopus_author.py]: for me author_id, was number of type Long.
The if clause should cover this sitation.

**Comment by SilasK** [comment]
I changed the name of the Notebook

File: scopus/scopus_api.py (hunk 1 above)
Comment (jkitchin) [inline scopus/scopus_api.py]: that could be, they aren't quite the same, and the longer one is more specific.

**Pull request merged**